vendor/CMakeLists.txt

```diff
@@ -5,3 +5,36 @@ add_subdirectory(lapack)
 add_subdirectory(mini-gmp)
 add_subdirectory(pcg)
 add_subdirectory(plfit)
+
+# We add the infomap vendored target here, so that
+# we can keep infomap as a clean submodule.
+
+file(GLOB_RECURSE infomap_src CONFIGURE_DEPENDS "infomap/src/*.h" "infomap/src/*.cpp")
```

Comment:
I'm not really a fan of globs in makefiles because build systems often cannot detect when new files are added. It is probably okay here as it is a vendored project, but I might replace this with an explicit list of files instead later.